CHANGELOG.md

```diff
@@ -2,69 +2,71 @@
 
 ## What's New
 
-* Converged Tunneler/Router
 * Transwarp beta_1
+* Converged Tunneler/Router (Beta)
+    * Updates to the host.v1 config type
+    * New host.v2 config type
+    * New health events
 
 ## Fixes
+
 * Service Policies with no Posture Checks properly grant access even if an identity has another Service Policies with posture checks
 * Removing entities with no referencing entities via attributes no longer panics
 * API Sessions and Current Api Session now share the same modeling logic
 * API Sessions now have lastActivityAt and cachedLastActivityAt
 * Enrolling and unenrolling in MFA sets MFA posture data
 
+## Transwarp beta_1
 
-## Converged Tunneler/Router
+See the Transwarp beta_1 guide [here](doc/transwarp_b1/transwarp_b1.md).
 
-ziti-router can now run with the tunneler embedded. It has the same capabilities as ziti-tunnel. As
-ziti-tunnel gains new features, the combined ziti-router/tunnel should maintain feature parity.
+## Converged Tunneler/Router (Beta)
 
-## Transwarp beta_1
+ziti-router can now run with the tunneler embedded. It has the same capabilities as ziti-tunnel. As ziti-tunnel gains new features, the combined ziti-router/tunnel should maintain feature parity.
 
-See the Transwarp beta_1 guide [here](doc/transwarp_b1/transwarp_b1.md).
+This is a beta release. It should be relatively feature complete and bug-free for hosting services but intercept support is still nascent. Some features are likely to be changed based on user
+feedback.
+
+NOTE: The converged tunneler/router does not support running in tun mode. This mode may be deprecated for ziti-tunnel as well at some point in the future if no advantages over tproxy mode are evident.
```

Comment:
was any of the converged tunneler work tested in windows and do we need to say "linux only" (or rather "not windows")?

Reply:
I added some notes on OS compatibility